package.json

```diff
@@ -18,5 +18,9 @@
   "license": "MIT",
   "publishConfig": {
     "@addepar:registry": "https://registry.npmjs.org/"
+  },
+  "devDependencies": {
```

Comment:
should be true `dependencies`, otherwise they won't get included in the base app